sycl/test-e2e/Graph/RecordReplay/ext_oneapi_enqueue_functions_submit_with_event.cpp

```diff
@@ -52,8 +52,9 @@ int main() {
 
   auto GraphExec = Graph.finalize();
 
-  Queue.submit([&](handler &CGH) { CGH.ext_oneapi_graph(GraphExec); });
-  Queue.wait_and_throw();
+  exp_ext::submit_with_event(Queue, [&](handler &CGH) {
+    execute_graph(CGH, GraphExec);
```

Comment:
I'm curious why we don't need the `exp_ext` prefix here

Reply:
Haven't read this PR, but more often than not the answer is ADL.

Reply:
thanks for the pointer 👍